.build/__local__.json

```diff
@@ -0,0 +1 @@
+{"contractTypes":{},"manifest":"ethpm/3","sources":{}}
```

Comment:
These local settings should be added to the `.gitignore`, yeah?

Reply:
`build` is there, but not `.build`. Added to template for future propagation